config/locales/en.yml

```diff
@@ -146,8 +152,15 @@ en:
       lifecycle: Lifecycle
       alpha_release_notes: Alpha release notes
     provider:
+      confirm_offer: Check and confirm offer
+      confirm_reject: Check feedback and reject application
+      confirm_withdraw_offer: Check and confirm withdrawal
+      new_defer_offer: Check and confirm offer deferral
+      new_edit_response: Change response
+      new_offer: Conditions of offer
+      new_reject: Reject application
+      new_withdraw_offer: Withdraw offer
```

Comment:
Added localisations for all these pages, and aligned the key with the name of the view.